### PR TITLE
add sqlpp11-connector-sqlite/0.29

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **sqlpp11-connector-sqlite/0.29**

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/sqlpp11-connector-sqlite3/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+class sqlpp11Conan(ConanFile):
+    name = "sqlpp11-connector-sqlite3"
+    version = "0.29"
+    description = "A C++ wrapper for sqlite3 meant to be used in combination with sqlpp11."
+    settings = "os", "compiler", "build_type", "arch"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rbock/sqlpp11-connector-sqlite3"
+    license = "BSD 2-Clause"
```
Comment (Croydon) [inline recipes/sqlpp11-connector-sqlite3/all/conanfile.py]:
```suggestion
    license = "BSD-2-Clause"
```

**Comment by ericLemanissier** [comment]
log mentions compiler `14.10.25017`, which is outdated. `14.16.27023` does not have this problem.

**Comment by ericLemanissier** [comment]
this is (probably?) blocked by https://github.com/conan-io/conan-center-index/issues/645

**Comment by danimtb** [comment]
The VS versions in CI have been updated. Relaunching the CI

**Pull request merged**